WpNginxConfig.conf

```diff
@@ -0,0 +1,44 @@
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
+    include snippets/self-signed.conf;
```

Comment:
what are those snippets? where do we use them?

Reply:
The snippet folder contains two files one points the SSL Key and Certificate location. The other one defines SSL settings (strong ssl cipher suite)
So we tell to nginx configuration file to include them, so it can use them for the https conection.